tests/test_core.py

```diff
@@ -6585,7 +6585,7 @@ def test_demangle_stacks_symbol_map(self):
     # make sure the shortened name is the right one
     full_aborter = None
     short_aborter = None
-    for line in open('test_demangle_stacks.js.symbols').readlines():
```

Comment:
The test system already has read_file and read_binary helpers.   Maybe maybe a seperate change to update the tests?

What do you think about keeping the names `read_file` and `read_binary` .. I kind of prefer them over `read_text` and `read_bytes`.. which don't sound so much like file access as `read_file` and `read_binary` do.   When I see `read_file` I kind of know its going to take a filename `read_text` sounds more vague to me.

Reply:
That works. I changed everything to `read_file` and `read_binary`